linux/README.md

```diff
@@ -43,35 +44,15 @@ parent directory like so:
   └─ flutter-desktop-embedding/ (from https://github.com/google/flutter-desktop-embedding)
 ```
 
-Next you will need to build the flutter engine. First, after you've checked out
-the engine code, make sure to build the version appropriate for your checkout of
-Flutter. This can be done using the `engine.version` file. Here is how you can
-check out the correct version of the engine when inside the `engine/` repo.
+After you've checked out the engine code, make sure to build the version
+appropriate for your checkout of Flutter. This can be done using the
+`engine.version` file. Here is how you can check out the correct version of the
+engine when inside the `engine/` repo.
 
 ```
 $ git checkout $(cat ../flutter/bin/internal/engine.version)
 ```
 
-Then, follow the steps outlined on the flutter engine's
-[contributing](https://github.com/flutter/engine/blob/master/CONTRIBUTING.md)
-page so that you can build a copy of `libflutter_engine.so`.
```

Comment:
You'll still need a modified version of this; in addition to the 'gclient sync' concern in my other comment, there are steps like sudo ./build/install-build-deps.sh that need to be run as part of initial setup.

Reply:
Added!